### PR TITLE
Replace `update_repo_visibility` with `update_repo_settings`

### PR DESCRIPTION
**Pull request by ariG23498**
CC: @pcuenca

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ for model in iter(models):
 ## Update repo visibility
 
 ```python
-from huggingface_hub import update_repo_visibility
+from huggingface_hub import update_repo_setting
```
Comment (pcuenca) [inline README.md]:
```suggestion
from huggingface_hub import update_repo_settings
```

File: README.md [hunk 2]
```diff
@@ -19,12 +19,12 @@ for model in iter(models):
...
 
 models = ["some/repo"]
 
 def do_publish(source):
-    update_repo_visibility(source, private=False)
+    update_repo_setting(source, private=False)
```
Comment (pcuenca) [inline README.md]:
```suggestion
    update_repo_settings(source, private=False)
```

**Review by pcuenca — COMMENTED** [review]
I think it's `update_repo_settings`: https://github.com/huggingface/huggingface_hub/blob/v0.26.0/src/huggingface_hub/hf_api.py#L3648

**Comment by pcuenca** [comment]
ping @Vaibhavs10 to merge when he's back 🤗

**Review by Vaibhavs10 — APPROVED** [review]
Merci!

**Pull request merged**